forest/shared/src/macros.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::time::Duration;
+
+pub const DEFAULT_RETRIES: i32 = 3;
+pub const DEFAULT_DELAY: Duration = Duration::from_secs(60);
+
+#[macro_export]
+macro_rules! retry {
+    ($func:ident, $max_retries:expr, $delay:expr $(, $arg:expr)*) => {{
+        let mut retry_count = 0;
+        loop {
+            match $func($($arg),*).await {
+                Ok(val) => break Ok(val),
+                Err(err) => {
+                    retry_count += 1;
+                    info!("Retry attempt {} started with delay {:#?}.", retry_count, $delay);
```

Comment:
Should this be after the `break` line?

Reply:
I think `Retry attemp {} will start in {delay}s` would be more precise, given `sleep` happens after this line

Reply:
right, fixed